src/main/java/com/google/api/generator/gapic/composer/ServiceClientSampleCodeComposer.java

```diff
@@ -649,6 +693,136 @@ public static String composePagedCallableMethodHeaderSampleCode(
+                    .build())
+            .setBody(
+                Arrays.asList(
+                    CommentStatement.withComment(
+                        LineComment.withComment("Do something when receive a response."))))
+            .setReturnType(TypeNode.VOID)
+            .build();
+    MethodDefinition onErrorMethod =
+        MethodDefinition.builder()
+            .setIsOverride(true)
+            .setScope(ScopeNode.PUBLIC)
+            .setName("onError")
+            .setArguments(
+                VariableExpr.builder()
+                    .setVariable(
+                        Variable.builder()
+                            .setName("t")
+                            .setType(
+                                TypeNode.withReference(
+                                    ConcreteReference.withClazz(Throwable.class)))
+                            .build())
+                    .setIsDecl(true)
+                    .build())
+            .setBody(
+                Arrays.asList(
+                    CommentStatement.withComment(LineComment.withComment("Add error-handling"))))
+            .setReturnType(TypeNode.VOID)
+            .build();
+    MethodDefinition onCompletedMethod =
+        MethodDefinition.builder()
+            .setIsOverride(true)
+            .setScope(ScopeNode.PUBLIC)
+            .setName("onCompleted")
+            .setBody(
+                Arrays.asList(
+                    CommentStatement.withComment(
+                        LineComment.withComment("Do something when complete."))))
+            .setReturnType(TypeNode.VOID)
+            .build();
+    AnonymousClassExpr anonymousClassExpr =
+        AnonymousClassExpr.builder()
+            .setType(responseObserverType)
+            .setMethods(onNextMethod, onErrorMethod, onCompletedMethod)
+            .build();
+    AssignmentExpr responseObserverAssignmentExpr =
+        AssignmentExpr.builder()
+            .setVariableExpr(responseObserverVarExpr.toBuilder().setIsDecl(true).build())
+            .setValueExpr(anonymousClassExpr)
+            .build();
+
+    List<Expr> bodyExprs = new ArrayList<>();
+    bodyExprs.add(responseObserverAssignmentExpr);
+
+    // Create an assignment expression for request observer variable expression with invoking the
+    // client's streaming call.
+    // e.g. ApiStreamObserver<EchoRequest> requestObserver =
+    //          echoClient.collectCallable().clientStreamingCall(responseObserver);
+    TypeNode requestObserverType =
+        TypeNode.withReference(
+            ConcreteReference.builder()
+                .setClazz(ApiStreamObserver.class)
+                .setGenerics(method.inputType().reference())
+                .build());
+    VariableExpr requestObserverVarExpr =
+        VariableExpr.withVariable(
+            Variable.builder().setName("requestObserver").setType(responseObserverType).build());
+    MethodInvocationExpr clientStreamCallMethodInvocation =
```

Comment:
Nit: `clientStreamingCallMethodInvocationExpr` (ending with Expr) for consistency, or `clientStreamingCallInvocationExpr` for short.